### DISCUSSION TITLE
Open url page source directly in vscode

### DISCUSSION THREAD

**Discussion by steinrr** [microsoft/vscode-discussions#652] [Extension Development QnA] [answered]
I've been trying to find a way of opening web page source directly in Visual Studio Code (Mac) without using a browser's "view page source" or saving file and open.

I.e. just give the URL as a parameter and then get the source code of that page opened in Visual Studio Code.

Typically opening an URL that returns some html og json and have it open directly in an editor in vscode.

Is there some support for this in vscode - or any relevant extension that can do it?

**Comment by isidorn** [comment, marked as answer]
I am not sure if there is an extension for this, but yes it is possible to do it from an extension. So if none exist you could write one yourself.
